src/EnergyPlus/MixedAir.cc

```diff
@@ -4208,8 +4208,8 @@ void VentilationMechanicalProps::CalcMechVentController(
                     ZoneOAArea =
                         curZone.FloorArea * curZone.Multiplier * curZone.ListMultiplier * this->ZoneOAAreaRate(ZoneIndex) * curZoneOASchValue;
                     ZoneOAFlow = curZone.Multiplier * curZone.ListMultiplier * this->ZoneOAFlowRate(ZoneIndex) * curZoneOASchValue;
-                    ZoneOAACH = curZone.Multiplier * curZone.ListMultiplier *
-                                (this->ZoneOAACHRate(ZoneIndex) * state.dataHeatBal->Zone(ZoneIndex).Volume) * curZoneOASchValue / 3600.0;
+                    ZoneOAACH =
+                        curZone.Multiplier * curZone.ListMultiplier * (this->ZoneOAACHRate(ZoneIndex) * curZone.Volume) * curZoneOASchValue / 3600.0;
```

Comment:
Got it, so yeah, `curZone` is based on `ZoneNum`, `ZoneIndex` should not have been used here.  Works for me.